doc/source/dev/contributor/quickstart_gitpod.rst

```diff
@@ -0,0 +1,156 @@
+:orphan:
+
+.. _quickstart-gitpod:
+
+=======================================================
+Development environment quickstart guide (Gitpod)
+=======================================================
+
+This quick start guide covers:
+
+*  using GitPod for your SciPy development environment
+*  creating a personal fork of the SciPy repository on GitHub
+*  a quick tour of Gitpod and VSCode
+*  working on the SciPy documentation in SciPy
+
+Gitpod
+-------
+
+`Gitpod`_  is an open-source platform for automated and ready-to-code development environments. It enables developers to describe their dev environment as code and start instant and fresh development environments for each new task directly from your browser. This reduces the need to install local development environments and deal with incompatible dependencies.
+
+Gitpod GitHub integration
+--------------------------
+
+To be able to use Gitpod, you will need tohave the Gitpod app installed on your GitHub account, so if 
+you do not have an account yet, you will need to create one first. 
+
+Head over to the `Gitpod`_ website and click on the **Continue with GitHub** button. You will be redirected to the GitHub authentication page.
+You will then be asked to install the `Gitpod GitHub app <https://github.com/marketplace/gitpod-io>`_.
+
+Make sure to select **All repositories** access option to avoid issues with permissions later on. Click on the green **Install** button
+
+.. image:: ../../_static/gitpod/installing-gitpod-io.png 
+    :alt: Gitpod repository access and installation screenshot
+
+This will install the necessary hooks for the integration. 
+
+Forking the SciPy repository
+-----------------------------
+
+The best way to work on SciPy as a contributor is by making a fork of the repository first. 
+
+#. Browse to the `SciPy repository on GitHub`_ and `create your own fork`_.
+
+#. Browse to your fork. Your fork will have a URL like https://github.com/andyfaff/scipy, except with your GitHub username in place of "andyfaff".
+
+Starting Gitpod
+----------------
+Once you have authenticated to Gitpod through GitHub, you will see a **Gitpod** button next to the **Code** button on the repository.
+
+.. image:: ../../_static/gitpod/scipy-github.png 
+    :alt: SciPy repository with Gitpod button screenshot
+
+#. Click the "Gitpod" button. This will open a new tab on your web browser and start building your development environment. Please note this can take a few minutes.
```

Comment:
Can we add a note here that if the button doesn't show up, navigating to `https://gitpod.io/#https://github.com/USERNAME/scipy` will do the same thing?